.docker/Dockerfile-build

```diff
@@ -6,7 +6,7 @@ WORKDIR /go/src/github.com/ory/hydra
 RUN mkdir -p ./internal/httpclient
 
 COPY go.mod go.sum ./
-COPY internal/httpclient/go.* ./internal/httpclient
+COPY internal/httpclient/go.* ./internal/httpclient/
```

Comment:
Without a trailing slash, it will fail.

```
Step 6/21 : COPY internal/httpclient/go.* ./internal/httpclient
When using COPY with more than one source file, the destination must be a directory and end with a /
make: *** [docker] Error 1
```